docs/core/integration-quality-scale/rules/iqs010.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "IQS010 - Mark entity unavailable if appropriate"
+---
+
+IQS010 is as follows:
+
+Mark entity unavailable if appropriate
+
+## Reasoning
+
+If we can't fetch data from a device or service, we should mark it as unavailable.
+We do this to reflect a better state, than just showing the last known state.
+
+If we can successfully fetch data but are temporarily missing a few pieces of data, we should mark the entity state as unknown instead.
+
+## Example implementation
+
+Since there are many different ways this can be implemented, we will only provide the example for entities updating via `async_update` and for integrations using the coordinator.
```

Comment:
Always start with the coordinator in examples and when talking about this, since that's more common today.